src/textual/keys.py

```diff
@@ -200,11 +200,4 @@ class Keys(str, Enum):
     ShiftControlEnd = ControlShiftEnd
 
 
-KEY_BINDINGS = Keys._value2member_map_.values()
-
-
-@dataclass
-class Binding:
-    action: str
-    description: str
-    show: bool = False
+KEY_VALUES = frozenset(Keys._value2member_map_.values())
```

Comment:
[picky mode] I noticed that this dunder property of Enums is not officially supported by Python, contrary to `__members__` for example - wouldn't it be a bit safer to resolve this via a simple list comprehension?
> https://docs.python.org/3/library/enum.html#supported-dunder-names

Reply:
Yes, you're absolutely correct.